types/tinymce/index.d.ts

```diff
@@ -633,7 +633,7 @@ export interface Formatter {
 
   matchNode(node: html.Node, name: string, vars: {}, similar: boolean): {};
 
-  register(name: {}, format?: {}): void;
+  register(name: {} | string, format?: {} | object[]): void;
```

Comment:
Thanks for your comment Daniel.
I always thought as {} as the empty object, that could be used interchangeably as Object, as well. Why is it equivalent to string?